apps/frontend/src/modules/activity/libs/types/types.ts

```diff
@@ -4,4 +4,5 @@ export {
 	type ActivityLogGetAllItemResponseDto,
 	type ActivityLogGetAllResponseDto,
 	type ActivityLogQueryParameters,
+	type ProjectGetAllItemResponseDto,
```

Comment:
I'm not sure, but maybe it'll be better to import this type from `~/modules/projects/projects.ts` in places where you need it, and not reexport this here?